src/plugins/intel_cpu/src/nodes/embedding_segments_sum.h

```diff
@@ -34,16 +34,16 @@ class EmbeddingSegmentsSum : public Node, public EmbeddingBagSum {
 
 private:
     void initFromInputs() override;
-    void getIndices(int embIndex, const int*& indices, size_t& size, int& weightsIdx, bool& withWeight) override;
-    int32_t getNumSegments() const;
+    void getIndices(size_t embIndex, const int*& indices, size_t& size, int& weightsIdx, bool& withWeight) override;
+    size_t getNumSegments() const;
 
     static constexpr size_t SEGMENT_ID_IDX = 2lu;
     static constexpr size_t NUM_SEGMENTS_IDX = 3lu;
 
-    int lastNumSegments_ = 0;
+    size_t lastNumSegments_ = 0;
 
     const int* indices_ = nullptr;
-    const int* segmentIds_ = nullptr;
+    const size_t* segmentIds_ = nullptr;
```

Comment:
As it is described above, this filed should be a pointer to int32 data.

Reply:
Same to up.